ddl/column.go

```diff
@@ -379,6 +422,20 @@ func doModifyColumn(t *meta.Meta, job *model.Job, newCol *model.ColumnInfo, oldN
 	return ver, nil
 }
 
+// CheckForNullValue check for null values of the field.
+func CheckForNullValue(ctx sessionctx.Context, schema, table, oldCol, newCol model.CIStr) error {
+	sql := fmt.Sprintf(`select * from %s.%s where %s is null; `, schema.L, table.L, oldCol.L)
```

Comment:
It is better to add quote around table/schema name, in case there are names like `a b`.

Reply:
like \`a\`

Reply:
`select count(*) from %s.%s where %s is null;`, if the null values count is very large, this will oom.